CheckDrive.Web/CheckDrive.Web/ViewModels/DashboardViewModel.cs

```diff
@@ -25,4 +151,43 @@ public class SpliteChartData
     public decimal Ai95 { get; set; }
     public decimal Diesel { get; set; }
 }
+public class Debt
+{
+    public int Id { get; set; }
+    public string Driver { get; set; }
+
+    [Newtonsoft.Json.JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
```

Comment:
What is the purpose of this?

Reply:
This is needed to output the values ​​of the enums to the grid as text